src/legacy/ui/public/agg_types/controls/extended_bounds.tsx

```diff
@@ -0,0 +1,133 @@
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React, { useEffect } from 'react';
+
+import { EuiFieldNumber, EuiFlexGroup, EuiFlexItem, EuiFormRow, EuiIconTip } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { AggParamEditorProps } from 'ui/vis/editors/default';
+import { FormattedMessage } from '@kbn/i18n/react';
+
+interface Bounds {
+  min: string;
+  max: string;
+}
+
+function isBoundEmpty(value: string): boolean {
+  return value === '';
+}
+
+function areBoundsInvalid({ min, max }: Bounds): boolean {
+  if (isBoundEmpty(min) && isBoundEmpty(max)) {
+    return false;
+  }
+
+  const minValue: number = parseInt(min, 10);
+  const maxValue: number = parseInt(max, 10);
+
+  if (isNaN(minValue) || isNaN(maxValue)) {
+    return true;
+  }
+
+  return minValue > maxValue;
+}
+
+function ExtendedBoundsParamEditor({ value, setValue, setValidity }: AggParamEditorProps<Bounds>) {
+  const tooltipContent = i18n.translate('common.ui.aggTypes.extendedBoundsTooltip', {
+    defaultMessage:
+      'Min and Max do not filter the results, but rather extend the bounds of the result set',
+  });
+
+  const mainLabel = (
+    <>
+      <FormattedMessage
+        id="common.ui.aggTypes.extendedBoundsLabel"
+        defaultMessage="Extended bounds"
+      />{' '}
+      <EuiIconTip
+        position="right"
+        content={tooltipContent}
+        type="questionInCircle"
+        aria-label={tooltipContent}
+      />
+    </>
+  );
+
+  const minLabel = i18n.translate('common.ui.aggTypes.extendedBounds.minLabel', {
+    defaultMessage: 'Min',
+  });
+
+  const maxLabel = i18n.translate('common.ui.aggTypes.extendedBounds.maxLabel', {
+    defaultMessage: 'Max',
+  });
+
+  const isInvalid = areBoundsInvalid(value);
+  let error;
+
+  if (isInvalid) {
+    error = i18n.translate('common.ui.aggTypes.extendedBounds.errorMessage', {
+      defaultMessage: 'Min should be less than or equal to Max',
+    });
+  }
+
+  useEffect(
+    () => {
+      setValidity(!isInvalid);
+
+      // we reset validity when the element will be hidden
+      return () => setValidity(true);
+    },
+    [value]
+  );
+
+  return (
+    <EuiFormRow
+      fullWidth={true}
+      label={mainLabel}
+      isInvalid={isInvalid}
+      error={error}
+      className="visEditorSidebar__aggParamFormRow"
+    >
+      <EuiFlexGroup>
```

Comment:
Reduce spacing between via
```suggestion
      <EuiFlexGroup  gutterSize="s">
```